src/Illuminate/Foundation/Application.php

```diff
@@ -546,7 +546,7 @@ public function isLocal()
      */
     public function isProduction()
     {
-        return $this['env'] === 'production';
+        return ! in_array($this['env'], ['development', 'local', 'staging', 'testing']);
```

Comment:
Laravel only defines "local", "testing" and "production" as application environments. They are different from server environments. For example, I can deploy application with mode "production" in a staging server.

Reply:
I thought so too, until I looked into the database migration tests. There, someone named an environment 'development', thus I included it.